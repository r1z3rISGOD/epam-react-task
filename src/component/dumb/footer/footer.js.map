{"version":3,"file":"footer.js","sourceRoot":"","sources":["footer.tsx"],"names":[],"mappings":";;;AAAA,6BAA8B;AAC9B,yBAAsB;AAEf,IAAM,MAAM,GAAc;IAC/B,OAAO,CACD,gCAAQ,SAAS,EAAC,QAAQ;QACtB,8BAAM,SAAS,EAAC,cAAc,sBAAuB,CAChD,CACd,CAAA;AACH,CAAC,CAAA;AANY,QAAA,MAAM,UAMlB"}