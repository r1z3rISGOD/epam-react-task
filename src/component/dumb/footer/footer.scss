@import "./src/_variables/variables.scss";

.footer {
  align-items: center;
  display: flex;
  height: 100px;
  width: $width;
  background-color: #414141;
}

.footer__text {
  font-family: $font;
  font-size: 1.7rem;
  color: $color;
  padding-left: 75px;
}
