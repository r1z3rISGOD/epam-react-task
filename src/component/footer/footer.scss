$width: 100%;
$color: #e55f64;
$font: "Roboto", sans-serif;

.footer {
  align-items: center;
  display: flex;
  height: 100px;
  width: $width;
  background-color: #414141;
}

.footer__text {
  font-family: $font;
  font-size: 1.7rem;
  color: $color;
  padding-left: 75px;
}
