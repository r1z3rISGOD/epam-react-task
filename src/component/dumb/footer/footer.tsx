import * as React from 'react'
import './footer.scss'

export const Footer : React.FC = () => {
  return (
        <footer className="footer">
            <span className="footer__text">netflixroulette</span>
        </footer>
  )
}
