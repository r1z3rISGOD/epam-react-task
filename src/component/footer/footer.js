import React from "react";
import "./footer.scss"

export default function Footer() {
    return(
        <footer className="footer">
            <span className="footer__text">netflixroulette</span>
        </footer>
    )
}
