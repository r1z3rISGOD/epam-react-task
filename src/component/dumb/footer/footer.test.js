import { Footer } from './footer.tsx'
import React from 'react'

it('footer', () => {
  const comp = shallow(<Footer/>)
  expect(comp).toMatchSnapshot()
})
