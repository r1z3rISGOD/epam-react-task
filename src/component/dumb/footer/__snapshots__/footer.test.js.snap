// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`footer 1`] = `
<footer
  className="footer"
>
  <span
    className="footer__text"
  >
    netflixroulette
  </span>
</footer>
`;
